website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -58,7 +58,7 @@ apply to both Consul Enterprise and Consul community edition (CE).
 
 | Consul Version | Compatible Envoy Versions              |
 | -------------- | -------------------------------------- |
-| 1.18.x CE      | 1.28.1, 1.27.3, 1.26.7, 1.25.11        |
+| 1.18.x CE      | 1.28.2, 1.27.4, 1.26.8, 1.25.11        |
 | 1.17.x         | 1.27.3, 1.26.7, 1.25.11, 1.24.12       |
 | 1.16.x         | 1.26.7, 1.25.11, 1.24.12, 1.23.12      |
```

Comment:
Sorry @dduzgun-security , I overlooked this. Though I think you could argue this table format should change to be less redundant, we should update these as well since we're backporting the relevant version changes.
```suggestion
| 1.17.x         | 1.27.4, 1.26.8, 1.25.11, 1.24.12       |
| 1.16.x         | 1.26.8, 1.25.11, 1.24.12, 1.23.12      |
```